pkg/porter/cnab.go

```diff
@@ -122,42 +120,29 @@ func (o *sharedOptions) validateBundlePath() error {
 
 // defaultBundleFile defaults the bundle file to the bundle in the current directory
 // when none is set.
-func (o *sharedOptions) defaultBundleFile() error {
+func (o *sharedOptions) defaultBundleFile(cxt *context.Context) error {
 	if o.File != "" {
```

Comment:
In this PR I was just preserving the original logic. I'm not sure why I didn't validate if the file exists before. Possibly because I didn't have the context? We could do it now pretty easily.